rust/processor/src/models/default_models/parquet_move_resources.rs

```diff
@@ -0,0 +1,190 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(clippy::extra_unused_lifetimes)]
+
+use crate::{
+    parquet_processors::generic_parquet_processor::{HasVersion, NamedTable, SizeOf},
+    utils::util::standardize_address,
+};
+use anyhow::{Context, Result};
+use aptos_protos::transaction::v1::{
+    DeleteResource, MoveStructTag as MoveStructTagPB, WriteResource,
+};
+use field_count::FieldCount;
+use get_size::GetSize;
+use get_size_derive::GetSize;
+use parquet_derive::ParquetRecordWriter;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, FieldCount, GetSize, Serialize, ParquetRecordWriter)]
```

Comment:
does `GetSize` follow pointers, arc, etc? cc @just-in-chang

Reply:
I was using [Allocative](https://crates.io/crates/allocative) for IMC, and that seemed to work fine, but I haven't tested it extensively. Not too sure about `GetSize` tho

Reply:
ideally we can unify on one as we're doing the same thing

Reply:
True, allocative says they're "light weight". Not too sure if it's worth benchmarking to see which is faster tho. Also allocative is from Facebook 😃

Reply:
I think if ya'll just see which one is easier, or if they're the same go with the most updated one